src/tests/ports/catalog-utils.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ChainId, Network } from '@dcl/schemas'
+import { getQuerySources } from '../../ports/catalog/utils'
+import { getSubgraphNameForNetwork } from '../../subgraphUtils'
+
+const marketplaceChainId = 1
+const collectionsChainId = 137
+
+jest.mock('../../logic/chainIds', () => ({
+  getCollectionsChainId: () => collectionsChainId,
+  getMarketplaceChainId: () => marketplaceChainId,
+}))
+
+describe('getQuerySources', () => {
+  it('should return sources for a specific network', () => {
+    const filters = { network: Network.ETHEREUM }
+    const sources = getQuerySources(filters)
+    expect(sources).toEqual({
+      [Network.ETHEREUM]: getSubgraphNameForNetwork(
+        Network.ETHEREUM,
+        marketplaceChainId as ChainId
+      ),
+    })
+  })
```

Comment:
Would you mind writing the tests using a describe to identify the context of the execution of the test?
```suggestion
let filters: Filters (don't know the type)
let sources: Sources

describe('when filtering by network', () => {
 beforeEach(() => {
  filters = { network: Network.ETHEREUM }
  sources = getQuerySources(filters)
 })

  it('should return sources for the provided network', () => {
    expect(sources).toEqual({
      [Network.ETHEREUM]: getSubgraphNameForNetwork(
        Network.ETHEREUM,
        marketplaceChainId as ChainId
      ),
    })
  })
})
```